source/remediation_runbooks/scripts/enable_delivery_status_logging.py

```diff
@@ -79,13 +91,15 @@ def get_topic_attributes(topic_arn):
         return topic_attributes
 
     except Exception as e:
-        exit(f'Failed to get attributes of SNS topic '+topic_arn+': '+str(e))
+        exit(f'Failed to get attributes of SNS topic {topic_arn}: {str(e)}')
 
 def reset_to_recognized_state(topic_arn):
     """
     Used in case of error, will unset all delivery status logging parameters.
     """
     sns = connect_to_sns()
+
+    for endpoint in endpointTypes:
+        sns.set_topic_attributes(TopicArn=topic_arn, AttributeName=f'{endpoint}SuccessFeedbackRoleArn', AttributeValue='')
+        sns.set_topic_attributes(TopicArn=topic_arn, AttributeName=f'{endpoint}FailureFeedbackRoleArn', AttributeValue='')
```

Comment:
could probably use error handling here - if we're trying to reset, we should catch exceptions on each iteration, log, and continue